### DISCUSSION TITLE
About Sentiment Analysis

### DISCUSSION THREAD

**Discussion by HasiburFahim** [shahriar-rahman/A-Comparative-Analysis-of-Amazon-Book-Ratings-using-Collaborative-Filtering#18] [Q&A] [answered]
Hey Shahriar, are we planning on doing Sentiment or any Text processing?

**Comment by shahriar-rahman** [comment]
I'm planning on it, but need to complete the base model first.

**Reply by HasiburFahim** [reply]
Okay, but I think we should try some stuff in parallel.

**Reply by shahriar-rahman** [reply]
Absolutely, feel free to start working on it, it will help finish the research within the deadline.

**Reply by HasiburFahim** [reply]
Any ideas where to start? I mean any modules that I should look into?

**Reply by shahriar-rahman** [reply, marked as answer]
Best to start with the Sentiment Analyzer from nltk, the idea is to analyze the reviews and amalgamate the data into a hybrid score, which, in theory, should boost the performance.
Another start will be Spacy, which is a popular text processing package. 

Here are the links:
https://pypi.org/project/nltk/
https://pypi.org/project/spacy/
